### ISSUE TITLE
Accessibility testing for the Blazor Web template

### ISSUE THREAD

**Issue by mkArtakMSFT** [dotnet/aspnetcore#49548]
After we're done with the Blazor Web project template restructuring work #49412, and the Blazor Web Identity UI work in RC1, we need to run the accessibility insights over the new template and address all the issues.

**Comment by mkArtakMSFT** [comment]
This will need to be exuected after https://github.com/dotnet/aspnetcore/issues/48786 is complete.

**Comment by MackinnonBuck** [comment]
We fixed multiple accessibility issues in https://github.com/dotnet/aspnetcore/pull/50722, each of them relating to Identity UI:
* Multiple elements on the page with the same ID ("email")
* Color contrast issues:
  * External login page info text had insufficient contrast with the background
  * Link colors had insufficient contrast with the info alert background
  * Validation messages had insufficient contrast with the background
* An aria role issue where `<ul>` elements had the "alert" role (removed the "role" attribute from each `<ul>`)
* An invalid value for the `autocomplete` attribute (was "confirm-password", changed to "new-password").
